extensions/omni_txn/omni_txn.c

```diff
@@ -1,4 +1,5 @@
 // clang-format off
+#include "c.h"
```

Comment:
Minor: < instead of "

Was it required here? I thought it would come transitively below